vars/waitForHelixRuns.groovy

```diff
@@ -24,24 +24,26 @@ def call (def helixRunsBlob, String prStatusPrefix) {
     def failedRunMap = [:]
     def passed = true
 
-    for (int i = 0; i < helixRunsBlob.size(); i++) {
-        try {
-            def currentRun = helixRunsBlob[i];
-            def queueId = currentRun['QueueId']
-            def correlationId = currentRun['CorrelationId']
-            def statusUrl = "https://helix.dot.net/api/2017-04-14/jobs/${correlationId}/details"
-            def statusResponse = httpRequest consoleLogResponseBody: true, url: statusUrl
-            assert statusResponse != null
-            assert statusResponse.content != null
-            def statusContent = (new JsonSlurperClassic()).parseText(statusResponse.content)
-            def mcResultsUrl = "https://mc.dot.net/#/user/${getEncodedUrl(statusContent.Creator)}/${getEncodedUrl(statusContent.Source)}/${getEncodedUrl(statusContent.Type)}/${getEncodedUrl(statusContent.Build)}"
-            mcUrlMap[queueId] = mcResultsUrl
-        }
-        catch(Exception ex) {
-            passed = false
-            println(ex.toString());
-            println(ex.getMessage());
-            println(ex.getStackTrace());
+    // Wait until the Helix runs register as started. Wait up to 3 mins between checks
+    waitUntil (minRecurrencePeriod: 120, maxRecurrencePeriod: 180, unit: 'SECONDS') {
```

Comment:
Also is this enough to cause a fail state given that I removed https://github.com/dotnet/dotnet-ci/pull/709/files#diff-31d0c6c923774342db32d312ae4e4f1aL41 and https://github.com/dotnet/dotnet-ci/pull/709/files#diff-31d0c6c923774342db32d312ae4e4f1aL73?